tests/data/pre-h5.html

```diff
@@ -1,2 +1,3 @@
-<pre>PRE</pre>
+<pre>PRE
+  LINE2</pre>
```

Comment:
Please try to keep the test case minimal. This can easily be implemented as a separated test to verify multiline `pre`.

Also, changing the test suite in a refactoring PR reduces confidence about regressions, as the original test suite is not run against the new code.